.github/workflows/docker-publish.yml

```
@@ -101,13 +124,13 @@ jobs:
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
 
-      # - name: Check versions of the binaries used by Godwoken
-      #   run: |
-      # FIXME: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by godwoken)
-      #     docker run --rm godwoken-prebuilds godwoken --version
-      #     docker run --rm godwoken-prebuilds gw-tools --version
-      ###
-      #     docker run --rm godwoken-prebuilds ckb --version
-      #     docker run --rm godwoken-prebuilds ckb-cli --version
-      #     docker run --rm godwoken-prebuilds ckb-indexer --version
-      #     docker run --rm godwoken-prebuilds find /scripts
+      - name: Check versions of the binaries in ${{ fromJSON(steps.meta.outputs.json).tags[0] }}
+        env:
+          IMAGE: ${{ fromJSON(steps.meta.outputs.json).tags[0] }}
+        run: |
+          docker run --rm ${{ env.IMAGE }} godwoken --version
+          docker run --rm ${{ env.IMAGE }} gw-tools --version
+          docker run --rm ${{ env.IMAGE }} ckb --version
+          docker run --rm ${{ env.IMAGE }} ckb-cli --version
+          docker run --rm ${{ env.IMAGE }} ckb-indexer --version
+          docker run --rm ${{ env.IMAGE }} find /scripts
```

```suggestion
          docker run --rm ${{ env.IMAGE }} find /scripts -type f -exec sha1sum {} \;
```

https://github.com/cryptape/godwoken-internal/issues/206#issuecomment-1055421158